backend-opal/src/database/migrations/1574592520680_create_table_watchlists.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
-  up: `INSERT INTO watch_lists 
+  up: `INSERT INTO watch_list 
     (user_id, name, symbols) 
  VALUES 
     (1, 'adom', 'MSFT,EPAM'),
-    (1, 'adom', 'APPL,FB'),
-    (1, 'adom', 'GO,ALI'),
-    (1, 'adom', 'TENC,AMZN');`,
+    (1, 'adom', 'AAPL,FB'),
+    (1, 'adom', 'GO,BABA'),
+    (1, 'adom', 'JD,AMZN');`,
   down: `
-    DELETE FROM watch_lists 
+    DELETE FROM watch_list 
```

Comment:
@hondahmd this is not going to work with deployment.
you can't change existing migrations, that's the point, you need to change the existing database, not how it was constructed in the past

Reply:
this is like changing a commit's content in the past, it is doable, but very hard and everyone will have conflicts